doc/source/how-to/continuous-integration.rst

```diff
@@ -176,3 +176,354 @@ Workflow examples are provided for checking :ref:`Coding style`,
+  +-------------------------------------------------------------------------------------+------------------------+
+  | **read:gpg_key**                                                                    | |:x:|                  |
+  |  Read public user GPG keys                                                          |                        |
+  +-------------------------------------------------------------------------------------+------------------------+
+  | **admin:ssh_signing_key**                                                           | |:x:|                  |
+  |  Full control of public user SSH signing keys                                       |                        |
+  +-------------------------------------------------------------------------------------+------------------------+
+  | **write:ssh_signing_key**                                                           | |:x:|                  |
+  |  Write public user SSH signing keys                                                 |                        |
+  +-------------------------------------------------------------------------------------+------------------------+
+  | **read:ssh_signing_key**                                                            | |:x:|                  |
+  |  Read public user SSH signing keys                                                  |                        |
+  +-------------------------------------------------------------------------------------+------------------------+
+
+.. _pypi_tokens:
+
+PyPI tokens
+~~~~~~~~~~~
+
+.. _pypi_private_token:
+
+``PYANSYS_PYPI_PRIVATE_PAT``
+****************************
+
+Used for authentication when uploading libraries to the private Ansys PyPI index. 
+It can be used as password for the `twine <https://twine.readthedocs.io/en/stable/>`_ library.
+
+.. _pypi_token:
+
+``PYPI_TOKEN``
+**************
+
+The value of this token is unique for each repository. 
+Used for authentication when uploading libraries to the public PyPI index.
+
+
+``PYPI_TESTING_TOKEN``
+**********************
+
+Used for testing uploads to public PyPI index.
+
+
+Other tokens
+~~~~~~~~~~~~
+
+``HUGO_THEME_TOKEN``
+********************
+
+This token is used to ... # todo: to be added
+
+
+``TWINE_TOKEN``
+***************
+
+This token is used for authentication when uploading libraries to PyPI and its value might change
+across repositories.
+Depending if the library is uploaded to a public PyPI index or private, its value matches
+:ref:`pypi_private_token` or :ref:`pypi_token`.
+
+
+
+.. _organization_bot:
+
+Organization bot
+----------------
+
+Because the usage of user personal access tokens (PATs) is discouraged, ``Ansys`` and ``Ansys-internal``
+organizations provide with a bot called ``ansys-bot`` to perform certain tasks which requires autentification,
+for example github pages publication or docker image registry login.
```

Comment:
```suggestion
Because the usage of personal access tokens (PATs) is discouraged, the ``Ansys`` and ``Ansys-internal``
organizations provide a bot named ``ansys-bot`` to perform certain tasks that require authentication.
For example, this bot provides for publishing GitHub pages or logging into a Docker image registry.
```